client/src/components/job-application-component/box-row-component/index.jsx

```diff
@@ -0,0 +1,145 @@
+import * as React from "react";
+import { useState } from "react";
+import { Grid, Typography, Select, MenuItem, Box, FormControl, InputLabel } from "@mui/material";
+import DownloadIcon from "@mui/icons-material/Download";
+import { IMaskInput } from "react-imask";
+
+import PropTypes from "prop-types";
+import { BoxDivider } from "./index.styles";
+
+const TextMaskCustom = React.forwardRef(function TextMaskCustom(
+  { onChange, name, ...other },
+  ref,
+) {
+  return (
+    <IMaskInput
+      // eslint-disable-next-line react/jsx-props-no-spreading
+      {...other}
+      mask="(#00) 000-0000x0000"
+      definitions={{
+        "#": /[1-9]/,
+      }}
+      inputRef={ref}
+      onAccept={(value) => onChange({ target: { name, value } })}
+      overwrite
+    />
+  );
+});
+
+TextMaskCustom.propTypes = {
+  name: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired,
+};
+
+function BoxRowComponent({
+  leftSide,
+  rightSide,
+  downloadable,
+  editable,
+  setRightSide,
+  required,
+  options,
+}) {
+  const [selectedValue, setSelectedValue] = useState(rightSide[0] || "Select");
+
+  return (
+    <>
+      <Grid
+        container
+        justifyContent="space-between"
+        alignItems="center"
+        sx={{ mb: 1, paddingX: "20px" }}
+      >
+        <Grid item xs={4} container alignItems="center">
+          <span>
+            <Typography variant="body1" align="left">
```

Comment:
For the left side variable, are you able to make it aligned to the top of the box, rather than the middle (vertically) when the field is "Application status"? 

Figma: 
![image](https://github.com/user-attachments/assets/6b5f3c45-87d8-449c-a88c-3917e8f1ec3d)

Yours:
![image](https://github.com/user-attachments/assets/86b90a9b-6cd3-4792-904d-37083bf0221a)